action.yml

```diff
@@ -48,16 +67,20 @@ inputs:
   warnings-as-errors:
     description: Fails the build if any warnings in *non test targets*.
     required: false
-    default: false
+    default: 'false'
```

Comment:
The quotes here seem incorrect.

Reply:
I agree it's weird. The schema interprets this value as a `string`.
https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

The validator built into VS Code complains if it isn't a `string`.
https://json.schemastore.org/github-action.json

Happy to remove this change if it's bothersome.